SashaSales/src/main/java/ua/artcode/daoSQL/implementations/UpDateCommands.java

```diff
@@ -0,0 +1,40 @@
+package ua.artcode.daoSQL.implementations;
+
+import org.apache.log4j.Logger;
+import ua.artcode.daoSQL.interfaces.UpdateDAO;
+import ua.artcode.util.ConnectionFactory;
+import ua.artcode.util.PropertiesHolder;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Created by work on 13.11.2016.
+ */
+public class UpDateCommands implements UpdateDAO {
+
+    private static final Logger LOGGER = Logger.getLogger(UpDateCommands.class);
+
+    public static final String UPDATE_STUDENT_BY_GROUP = "UPDATE student SET group_id = %d WHERE id = '%s';";
+
+    public boolean updateStudentByGroup(int student_id, int group_id) throws ClassNotFoundException {
+
+        Class.forName(PropertiesHolder.getProperty("CLASS_NAME_JDBC_DRIVER"));
```

Comment:
Load a driver class onlu once, put it to ConnectionHolder -> Factory for Connections